src/main/kotlin/uk/gov/justice/hmpps/offendersearch/dto/MatchRequest.kt

```diff
@@ -0,0 +1,18 @@
+package uk.gov.justice.hmpps.offendersearch.dto
+
+import io.swagger.annotations.ApiModelProperty
+import java.time.LocalDate
+import javax.validation.constraints.NotBlank
+import javax.validation.constraints.Past
+
+data class MatchRequest(
+    @ApiModelProperty(value = "Offender first name", example = "john",  position = 1) val firstName: String? = null,
+    @field:NotBlank(message = "Surname is required")
+    @ApiModelProperty(value = "Offender surname", example = "smith", position = 2) val surname: String? = null,
```

Comment:
do you need a default `null` value if the field is required

Reply:
agreed, I'll fix